astroquery/ibe/core.py

```diff
@@ -24,16 +26,14 @@
 
 __all__ = ['Ibe', 'IbeClass']
 
-
 class IbeClass(BaseQuery):
     URL = conf.server
     MISSION = conf.mission
-    DATASET = conf.dataset
     TABLE = conf.table
     TIMEOUT = conf.timeout
 
     def query_region(
-            self, coordinate=None, where=None, mission=None, dataset=None,
```

Comment:
Removing kwargs, even if they are unused, should be done via a deprecation. 
There is a decorator for doing that in astropy, and it was backported to the LTS release (2.0.12), too, but not to 3.0.x, so I suppose we shouldn't yet use it here. 
@keflavich - what do you think? In astroML I worked around this very same issue by copying the decorator to `astroML.utils` and importing it from there when the astropy version is not the preferred one (we also do similar things in `photutils` to work around issues that are not available in all supported versions of astropy).

Reply:
I'm onboard with using those decorators, but in this case, I think these kwargs are being removed because they didn't do anything?  But your point below that it's breaking the tests suggests that yes, we should use the deprecation and copy over the deprecator decorator.

Reply:
I'll open a PR for those then tomorrow during the co-working hour.